### ISSUE TITLE
RC Test: gravity-force-lab-basics:1.0.0-rc.1

### ISSUE THREAD

**Issue by zepumph** [phetsims/QA#429]
@terracoda, @emily-phet, @jessegreenberg, gravity-force-lab-basics:1.0.0-rc.1 is ready for RC testing. This is being published for the first time, and includes sound as well as PDOM descriptions. Document issues in https://github.com/phetsims/gravity-force-lab-basics/issues and link to this issue.

Assigning to @ariel-phet for prioritization.

<details>
<summary><b>General RC Test</b></summary>

<h3>What to Test</h3>

- Click every single button.
- Test all possible forms of input.
  - Test all mouse/trackpad inputs.
  - Test all touchscreen inputs.
- If there is sound, make sure it works.
- Make sure you can't lose anything.
- Play with the sim normally.
- Try to break the sim.
- Test all query parameters on all platforms. (See [QA Book](https://github.com/phetsims/QA/blob/master/doc/qa-book.md) for a list of query parameters.)
- Download HTML on Chrome and iOS.
- Make sure the iFrame version of the simulation is working as intended on all platforms.
- Make sure the XHTML version of the simulation is working as intended on all platforms.
- Complete the test matrix.
- Don't forget to make sure the sim works with Legends of Learning.
- Check [this](https://docs.google.com/spreadsheets/d/1umIAmhn89WN1nzcHKhYJcv-n3Oj6ps1wITc-CjWYytE/edit#gid=0) LoL spreadsheet and notify AR or AM if it not there.
- If this is rc.1 please do a memory test.
- When making an issue, check to see if it was in a previously published version
- If there is a console available, check for errors and include them in the Problem Description.

<h3>Link(s)</h3>

- **[Simulation](https://phet-dev.colorado.edu/html/gravity-force-lab-basics/1.0.0-rc.1/phet/gravity-force-lab-basics_all_phet.html)**
- **[iFrame](https://phet-dev.colorado.edu/html/gravity-force-lab-basics/1.0.0-rc.1/phet/gravity-force-lab-basics_all_iframe_phet.html)**
- **[XHTML](https://phet-dev.colorado.edu/html/gravity-force-lab-basics/1.0.0-rc.1/phet/xhtml/gravity-force-lab-basics_all.xhtml)**
- **[Test Matrix](https://docs.google.com/spreadsheets/d/14SmRPdAdphYfSf-7n0gR2iltcZUHmnQqsmGKd8hTBYg/edit#gid=2013811824)**
- **[Legends of Learning Harness](https://developers.legendsoflearning.com/public-harness/index.html?startPayload=%7B%22languageCode%22%3A%22en%22%7D)**

<hr>

</details>

<details>
<summary><b>Accessibility (a11y) RC Test</b></summary>

<h3>What to Test</h3>

- Specific instructions can be found below.
- Make sure the a11y feature doesn't negatively affect the sim in any way.
- Test all possible forms of input.
  - Test all mouse/trackpad inputs.
  - Test all touchscreen inputs.
  - Test all keyboard navigation inputs (if applicable).
  - Test all forms of input with a screen reader (if applicable).

<h3>Link(s)</h3>

- **[a11y View](https://phet-dev.colorado.edu/html/gravity-force-lab-basics/1.0.0-rc.1/phet/gravity-force-lab-basics_a11y_view.html)**
- **[Simulation](https://phet-dev.colorado.edu/html/gravity-force-lab-basics/1.0.0-rc.1/phet/gravity-force-lab-basics_all_phet.html)**
- **[Test Matrix](https://docs.google.com/spreadsheets/d/14SmRPdAdphYfSf-7n0gR2iltcZUHmnQqsmGKd8hTBYg/edit#gid=2013811824)**

<h3>Screen Readers</h3>

This sim supports screen readers. If you are unfamiliar with screen readers, please ask Katie to introduce you to screen readers. If you simply need a refresher on screen readers, please consult the [QA Book](link), which should have all of the information you need as well as a link to a screen reader tutorial made by Jesse. Otherwise, look over the a11y view before opening the simulation. Once you've done that, open the simulation and make sure alerts and descriptions work as intended.

<h3>Platforms and Screen Readers to Be Tested</h3>

- [x] Windows 10 + Latest Chrome + Latest JAWS
- [x] Windows 10 + Latest Firefox + Latest NVDA
- [x] macOS + Safari + VoiceOver
- [x] iOS + Safari + VoiceOver (only if specified in testing issue)

<h3>Critical Screen Reader Information</h3>

We are tracking known screen reader bugs in [this Google Document](https://drive.google.com/open?id=12kTs-g-iKEIH1dyG7Q41_W_oNL4gUKbkW-IQgZjMUBw). If you find a screen reader bug, please check it against this list.

<h3>Keyboard Navigation</h3>

This sim supports keyboard navigation. Please make sure it works as intended on all platforms by itself and with a screen reader.

<h3>Final Requests</h3>

- [x] If this sim is in [this list](https://github.com/phetsims/perennial/blob/master/data/accessibility), make an issue in [website](https://github.com/phetsims/website) to ask if PhET research page links need updating. Please assign to @terracoda and @emily-phet.

<hr>

</details>

<details>
<summary><b>FAQs for QA Members</b></summary>

<br>
  
  

  <details>
  <summary><i>There are multiple tests in this issue... Which test should I do first?</i></summary> 
  
  Test in order! Test the first thing first, the second thing second, and so on.

  </details>

  <br>

  

  <details>
  <summary><i>How should I format my issue?</i></summary>
  
  Here's a template for making issues:

      <b>Test Device</b>

      blah

      <b>Operating System</b>

      blah

      <b>Browser</b>

      blah

      <b>Problem Description</b>

      blah

      <b>Steps to Reproduce</b>

      blah

      <b>Visuals</b>

      blah

      <details>
      <summary><b>Troubleshooting Information</b></summary>

      blah

      </details>

  </details>

  <br>

  

  <details>
  <summary><i>Who should I assign?</i></summary>
  
  We typically assign the developer who opened the issue in the QA repository.

  </details>
  
  <br>

  

  <details>
  <summary><i>My question isn't in here... What should I do?</i></summary>
  
  You should:

  1. Consult the [QA Book](https://github.com/phetsims/QA/blob/master/doc/qa-book.md).
  2. Google it.
  3. Ask Katie.
  4. Ask a developer.
  5. Google it again.
  6. Cry.

  </details>

<br>

<hr>

</details>

**Comment by KatieWoe** [comment]
Memory test results look good
![memresult](https://user-images.githubusercontent.com/41024075/65187782-eca10280-da29-11e9-82d3-8df1a336a795.png)

**Comment by KatieWoe** [comment]
https://github.com/phetsims/resistance-in-a-wire/issues/177 Occurs quite obviously when describing the mass spinners.

**Comment by KatieWoe** [comment]
QA is done

**Comment by zepumph** [comment]
Thanks @KatieWoe! Closing

**Comment by jessegreenberg** [comment]
@KatieWoe said
> phetsims/resistance-in-a-wire#177 Occurs quite obviously when describing the mass spinners.

It looks like the issue is a bit different. The min value of the slider is 1000000000 and JAWS reports this as "1 zero zero zero zero zero zero zero ..." instead of reading "1 billion". Does this require further investigation?

**Comment by terracoda** [comment]
@zepumph, is it possible to use 1 and 10 for the min and max values for the mass sliders instead of "1 000 000 000" and "10 000 000 000"?

Would that mess things up?

**Comment by terracoda** [comment]
I'll close this as we will investigate further in https://github.com/phetsims/gravity-force-lab-basics/issues/181
